assets/jsons/generosity_challenge/chat_scripts/chat_script_day_0.json

```diff
@@ -47,51 +47,57 @@
-                              "next": {
+                          "type": "textMessage",
+                          "side": "interlocutor",
+                          "text": "I need to let you know how great a job you are doing and notify you to keep going. Okay?",
+                          "next": {
+                            "type": "buttonGroupAnswer",
+                            "side": "user",
+                            "options": [
+                              {
                                 "type": "textMessage",
-                                "side": "interlocutor",
-                                "text": "That's ok. I'll still try and help as much as I can. Let me show you what I found walking round the city today",
+                                "side": "user",
+                                "text": "Not right now",
+                                "answerText": "Not right now",
+                                "amplitudeEvent": "custom_event_demo_name",
                                 "next": {
-                                  "branch": "image01"
+                                  "type": "textMessage",
+                                  "side": "interlocutor",
+                                  "text": "That's ok. I'll still try and help as much as I can. Let me show you what I found walking round the city today",
+                                  "next": {
+                                    "branch": "image01"
+                                  }
                                 }
-                              }
-                            },
-                            {
-                              "type": "textMessage",
-                              "side": "user",
-                              "text": "Yes",
-                              "amplitudeEvent": "generosity_challenge_chat_user_action",
-                              "answerText": "Yes",
-                              "functionName": "pushNotificationPermission",
-                              "next": {
+                              },
+                              {
                                 "type": "textMessage",
-                                "side": "interlocutor",
-                                "text": "Awesome! Let me show you what I found walking round the city today",
+                                "side": "user",
+                                "text": "Yes",
+                                "amplitudeEvent": "generosity_challenge_chat_user_action",
+                                "answerText": "Yes",
+                                "functionName": "pushNotificationPermission",
                                 "next": {
-                                  "branch": "image01"
+                                  "type": "textMessage",
+                                  "side": "interlocutor",
+                                  "text": "Awesome! Let me show you what I found walking round the city today",
```

Comment:
"walking round" > "walking around"

Reply:
existing typo, will fix

Reply:
actually no i wont change this, its existing text and it is correct, just less formal.